addons/website_event/controllers/main.py

```diff
@@ -70,7 +70,16 @@ def events(self, page=1, **searches):
         events = events[(page - 1) * step:page * step]
 
         # count by domains without self search
-        domain_search = [('name', 'ilike', fuzzy_search_term or searches['search'])] if searches['search'] else []
+        country_or_city_event_ids = Event.sudo().search([
```

Comment:
Note that we now have the ``address_search`` field thanks to @pko-odoo that searches on all "interesting" address fields. Couldn't we use it so that frontend and backend address searches are the same ? Moreover it can be directly included in domain_search, which means probably less queries in performance tests :)

Reply:
Hello @tde-banana-odoo 

We've utilized the address_search field, but we can't directly use it within domain_search because it won't be accessible for public user 🙂

Reply:
And globally having `_search_get_detail()` return `'requires_sudo': True,` would bypass other restrictions.